### ISSUE TITLE
LogUnits are not removed from the layout when declared unresponsive.

### ISSUE THREAD

**Issue by rogermichoud** [CorfuDB/CorfuDB#1036]
## Overview

When a logunit is detected as unresponsive, we add it to the unresponsive section of the layout but we don't remove it from the new logUnits new segment.

-  **Bug report.** 
Bug was found in addNodeCmdlet branch.

**Comment by zalokhan** [comment]
Fix by #1026

**Comment by annym** [comment]
@no2chem We should merge this asap, it is blocking us from testing cluster node kill. This bug prevents test to continue right after a node in the cluster is killed.

**Comment by no2chem** [comment]
@annym Okay - I've given my approval already. I was told that the PR is under review still, but let's proceed with the merge.

@zalokhan , can you rebase and fix the conflict? Once that is done we can merge it in.

**Comment by zalokhan** [comment]
@no2chem #1026 still requires one final review pass.
To unblock this issue, I created a separate PR #1092

**Comment by annym** [comment]
Thanks, it was better to isolate that change! @zalokhan 
@no2chem Already reviewed PR #1092
